content/pages/startComputeDataset.json

```
@@ -18,6 +18,12 @@
         "options": [],
         "sortOptions": false
       }
-    ]
+    ],
+    "termsAndConditions": {
+      "name": "termsAndConditions",
+      "type": "checkbox",
+      "options": ["I agree to the"],
+      "postfixText": " Terms and Conditions"
+    }
```

suggestion: to make this re-usable in probable future scenarios, I'd propose the following approach:

```suggestion
    "termsAndConditions": {
      "name": "termsAndConditions",
      "type": "checkbox",
      "options": ["Terms and Conditions"],
      "prefix": ["I agree to the"],
      "action": ["/terms"]
    }
```

This would enable a few scenarios:
```json
    "termsAndConditions": {
      "name": "termsAndConditions",
      "type": "checkbox",
      "options": ["Terms and Conditions", "Privacy Policy"],
      "prefix": ["I agree to the", "I have read and accept the"],
      "postfix": ["for sure!"],
      "action": ["/terms", "/privacy"]
    }
```

---

So `prefix` and `postfix` would just be part of the label and the `actions` would be attached to the text inside `options`?